recipes/ectyper/meta.yaml

```diff
@@ -0,0 +1,33 @@
+about:
+    license: MIT
+    summary: Ectyper is a python script for serotyping e.coli genomes
+    author: Camille La Rose, Chad Laing, Sam Sung
+    home: https://github.com/phac-nml/ecoli_serotyping
+
+package:
+    name: ectyper
+    version: 0.1.0
```

Comment:
Now the number is still not the number from the github release. It should be 0.1 isn't it?
You could also use template vars for the version number, this way an upgrade is easier later on.